### PR TITLE
[ENHANCEMENT] based on alpine docker image, much smaller.

### PR DESCRIPTION
**Pull request by Doublemine**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 COPY ./dist/ /usr/share/nginx/html/
 COPY ./nginx.conf /etc/nginx/nginx.conf
 COPY ./run.sh /usr/share/run.sh
 
 ENV TZ=Asia/Shanghai
 
-RUN chmod +x /usr/share/run.sh 
+RUN set -ex \
+    && apk add --no-cache bash \
+    && chmod +x /usr/share/run.sh
```
Comment (JaredTan95) [inline Dockerfile]: Could you help to adding `    && cp /usr/share/zoneinfo/Asia/Shanghai /etc/localtime \
    && echo "Asia/Shanghai" > /etc/timezone \` here?

File: Dockerfile-ci [hunk 2]
```diff
@@ -5,14 +5,17 @@ RUN cd /app/ \
     && npm install \
     && npm run build
 
-FROM nginx:1.14
+FROM nginx:1.14-alpine
 COPY --from=node-builder /app/ /app/
-RUN mv /app/dist/* /usr/share/nginx/html/ \
+RUN set -ex \
+    && apk add --no-cache bash \
+    && mv /app/dist/* /usr/share/nginx/html/ \
```
Comment (JaredTan95) [inline Dockerfile-ci]: Is this method of copying from `node-builder` works?

**Comment by wu-sheng** [comment]
Please don't add any time zone setting in dockerfile. We need to keep UI suitable for the global. 

Also, after we moved this, the docker file may be not needed at all. Because it can't be released independently.

**Comment by JaredTan95** [comment]
> Please don't add any time zone setting in dockerfile. We need to keep UI suitable for the global.
> 
> Also, after we moved this, the docker file may be not needed at all. Because it can't be released independently.

Got it, two options:

1、we use the default time zone in dockerfile temporarily. 
2、delete all dockerfile in this repo, in order to transfer it to apache-skywalking-rocketbot repo.

@wu-sheng I prefer 2ed. How about @TinyAllen ?

**Comment by Doublemine** [comment]
LGTM, I'll close this PR.

**Comment by TinyAllen** [comment]
> LGTM, I'll close this PR.

You can reopen it and delete the Dockerfile😄

**Comment by Doublemine** [comment]
Okay😂

**Comment by wu-sheng** [comment]
UI in SkyWalking is binding release with webapp, also docker file released as UI too. So no need to add more, we just need to change the main repo submodule config.

**Comment by JaredTan95** [comment]
> UI in SkyWalking is binding release with webapp, also docker file released as UI too. So no need to add more, we just need to change the main repo submodule config.

I knew, but we think this repo can startup alone with nginx. So, we hope to provide some helpful dockerfile somewhere or not.

**Comment by wu-sheng** [comment]
Docker file could be hosted in here, but we can't release them, and also need to highlight the purpose of dockerfile is just for developer review.

**Comment by JaredTan95** [comment]
> > UI in SkyWalking is binding release with webapp, also docker file released as UI too. So no need to add more, we just need to change the main repo submodule config.
> 
> I knew, but we think this repo can startup alone with nginx. So, we hope to provide some helpful dockerfile somewhere or not.

 Okay, we decide to hold those dockerfile in this repo.  But we need to remove time zone settings and adding some description in `README.md` to highlight the dockerfile just for developer review  or helpful suggestions. @Doublemine  Could you reopen this pr and fix it? Thanks~

File: README.md [hunk 3]
```diff
@@ -100,6 +100,21 @@ docker run -p 8080:80 -d -e SKYWALKING_URL=127.0.0.1:1234,127.0.0.1:1235 rocketb
...
+```dockerfile
+ls /usr/share/zoneinfo
+cp /usr/share/zoneinfo/Asia/Shanghai /etc/localtime
+```
+
+Now specify your timezone
+
+```
+echo "Asia/Shanghai" >  /etc/timezone
+```
```
Comment (JaredTan95) [inline README.md]: And needs some description here, such as `Those dockerfile is just for developer review or refer methods.` etc.

**Pull request merged**